pyqtgraph/graphicsItems/ErrorBarItem.py

```diff
@@ -82,23 +85,27 @@ def drawPath(self):
                     y2 = y
                 else:
                     y2 = y + top
+
+            y1_y2 = np.column_stack((y1, y2)).flatten()
+            y2s = np.column_stack((y2, y2)).flatten()
```

Comment:
nit: move these (`y1s`, `y2s`, `x1s`, `x2s`) to just where they're needed to avoid creating them if there aren't beams